test/test_log.rb

```diff
@@ -411,6 +411,50 @@ def test_different_message
     end
   end
 
+  sub_test_case "ignore_same_log_interval" do
+    teardown do
+      Thread.current[:last_same_log] = nil
+    end
```

Comment:
Should we add `Timecop.return` in `#teardown`?

Reply:
`Timecop.return` is called in parent teardown.

Reply:
Oh! I got it.